tests/qm-sanity-test/test.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+source ../e2e/lib/utils
+
+# Verify qm is up and running
+check_qm_is_up(){
+    if [ "$(systemctl is-active qm)" == "active" ]; then
+        info_message "check_qm_is_up(): qm is active"
+    else
+        info_message "check_qm_is_up(): qm is not active"
+        exit 1
+    fi
+}
+
+# Verify bluechi nodes are connected
+check_bluechi_nodes_connected(){
+    expected_result="Connected to controller as 'qm.localrootfs'"
+    if [[ "$(podman exec -it qm bash -c "systemctl status bluechi-agent" \
```

Comment:
It is better to check that bluechictl is not broken
Advise to use `bluechictl status` inside the host.
should return two nodes connected 

Suggestion, 
check_bluechi_is_ok
see my comment below
systemctl is-active bluechi-controller
systemctl status show 
It can replace the checks inside the method